packages/gatsby-source-wordpress/src/gatsby-node.js

```diff
@@ -68,44 +68,51 @@ exports.sourceNodes = async (
     url = `${_siteURL}/wp-json`
   }
 
-  console.log()
-  console.log(
-    colorized.out(
-      `=START PLUGIN=====================================`,
-      colorized.color.Font.FgBlue
+  if (_verbose) console.log()
```

Comment:
plugins are now passed a `reporter` object for doing this, if you want to make use of it. e.g. reporter.verbose('blah')